### PR TITLE
WFLY-19161: add documentation for configuration file aliases

### PR DESCRIPTION
**Pull request by michpetrov**
Issue: [WFLY-19161](https://issues.redhat.com/browse/WFLY-19161)

____

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

### CODE REVIEW

File: docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc [hunk 1]
```diff
@@ -161,11 +161,34 @@ standard configuration files, e.g. `standalone-full.xml`,
...
+
+* `bin/standalone.sh -c ha`
+
+.Available aliases
+|===
+| Configuration file | Aliases
+
+| standalone-full.xml     | full
+
+| *-ha.xml                | ha
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc:181]: @michpetrov What does '*' mean here and in the others?

**Comment by bstansberry** [comment]
@michpetrov Adding this content in https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/Getting_Started_Guide.adoc#starting-wildfly-with-an-alternate-configuration would be good too.

**Comment by darranl** [comment]
Adding 32.x as we need the doc in Final, not adding other labels as not strictly necessary for Beta.

File: docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc (hunk 1 above)
Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc:181]: @michpetrov Why not just spell it out? Any symbol people will try and interpret what it means and may come up with novel interpretations, or will just wonder why they're needing to use mental energy.

**Comment by bstansberry** [comment]
> @bstansberry I assume a link from that section to this one will do?

OK, that works. That section doesn't list all the available files, so copying in this content that does list them all would be confusing.

File: docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc (hunk 1 above)
Comment (michpetrov) [reply docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc:181]: @bstansberry well, it's superfluous information and AFAIK it's not good UX to have each line start with the same word, but I'm not a docs person so if you don't mind it I'll keep it there

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @michpetrov!

**Pull request merged**